.rspec

```diff
@@ -1,3 +1,4 @@
 --format documentation
 --color
 --tty
+--fail-fast
```

Comment:
please remove `fail-fast` from this PR as it's not what we want when running rspec tests